exec/java-exec/src/test/java/org/apache/drill/test/rowSet/RowSetSchema.java

```diff
@@ -83,7 +85,13 @@ private void updateStructure(int index, PhysicalSchema children) {
     public boolean isMap() { return mapSchema != null; }
     public PhysicalSchema mapSchema() { return mapSchema; }
     public MaterializedField field() { return field; }
-    public String fullName() { return fullName; }
+
+    /**
+     * Returns full name path of the column.
```

Comment:
This change is fine, but the code here is removed/replaced in PR #914. I believe that #914 correctly follows the guidelines you have created here, but feel free to take a look to check if I missed anything.